tests/integration/cartography/intel/aws/inspector/test_inspector.py

```diff
@@ -5,8 +5,11 @@
 from tests.data.aws.inspector import LIST_FINDINGS_NETWORK
 
 TEST_UPDATE_TAG = 123456
+TEST_ACC_ID_1 = '123456789011'
+TEST_ACC_ID_2 = '123456789012'
 
 
+# TODO delete this redundant test after #1088 is closed. I am keeping this for now to show that no existing tests break.
```

Comment:
Will fast follow with a PR to remove these tests since they are superceded by test_inspector_sync.py.